calico_node/filesystem/sbin/check_readiness

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Check felix readiness
+if [ ! -z "$FELIX_CHECKREADINESS" ]; then
+    if [ ! /usr/bin/wget localhost:9099/readiness -O - ]; then
```

Comment:
When k8s calls the readiness endpoint, it checks that 200 <= RC <= 400. We should probably do the same here, but maybe wget is already doing that check and resolving it for us?

Also probably worth being explicit with a timeout via `-T` - by default wget will timeout after 900 seconds, which is a bit long for this check. Probably no more than 10 seconds given we configure k8s to call this script every 10s?